packages/twenty-front/src/modules/prefetch/components/PrefetchDataProvider.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import { PrefetchRunQueriesEffect } from '@/prefetch/components/PrefetchRunQueriesEffect';
+
+export const PrefetchDataProvider = ({ children }: React.PropsWithChildren) => {
+  return (
+    <>
+      <PrefetchRunQueriesEffect />
```

Comment:
Why don't we just put this Effect component into the hierarchy ?

Reply:
it's to avoid polluting the main index.tsx but I don't have a strong opinion